### PR TITLE
ref: draggable flatlist source

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -65,6 +65,7 @@
         "react-native-default-preference": "1.4.4",
         "react-native-device-info": "8.7.1",
         "react-native-document-picker": "https://github.com/BlueWallet/react-native-document-picker#301c551970fd053d9f1b99a05e2236210ad9dcf8",
+        "react-native-draggable-flatlist": "github:BlueWallet/react-native-draggable-flatlist#ebfddc4",
```
Comment (Overtorment) [inline package-lock.json:68]: Is that a brand new package of yours or it's your fork of someone's package?

**Review by Overtorment — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (marcosrdz) [reply package-lock.json:68]: We have been using this package for a while now. This is to fork it to the org for better management of updates

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/mcupvkjof7xnri443g47fjr3cy?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/11395)

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**